### PR TITLE
Offload issues to prevent memory issues with large collections

### PR DESCRIPTION
**Pull request by jasper-jager**
Issues will be offloaded to file, a byte pointer will be saved to keep track of merged issues

### CODE REVIEW

File: gobcore/logging/logger.py [hunk 1]
```diff
@@ -100,25 +106,86 @@ def __init__(self, name=None):
...
 
+    def write_issue(self, issue):
+        if not self._offload_empty:
+            self._offload_file.write("\n")
+
+        # Store the position in the file before the issue and return it to store a pointer
+        issue_offset = self._offload_file.tell()
+
+        self._offload_file.write(issue.json)
+        self._offload_empty = False
```
Comment (JKoetsier) [inline gobcore/logging/logger.py:172]: Why not always write _issue + "\n"_? Removes the _offload_empty variable and check.

Comment (jasper-jager) [reply gobcore/logging/logger.py:172]: Ah true, a remainder of when I was first writing the file to be able to be parsed by ijson. Will fix

File: gobcore/logging/logger.py [hunk 2]
```diff
@@ -100,25 +106,86 @@ def __init__(self, name=None):
...
+            # First close the open file since no more issues will be added
+            self.close_offload_file()
+
+            with open(self._offload_filename, 'r') as offload_file:
+                issue_byte_positions = list(self._issues.values())
+                issue_byte_positions.sort()
+                for issue_position in issue_byte_positions:
+                    offload_file.seek(issue_position)
+                    issue = json.loads(offload_file.readline().rstrip("\n"))
+                    yield issue
```
Comment (JKoetsier) [inline gobcore/logging/logger.py:188]: Nice solution, keeping the old issues in place but inaccessable :+1:

File: gobcore/quality/issue.py [hunk 3]
```diff
@@ -242,8 +280,12 @@ def is_functional_process(process):
 def process_issues(msg):
     issues = logger.get_issues()
 
+    header = msg.get('header', {})
+
+    print(QualityUpdate.CATALOG)
```
Comment (JKoetsier) [inline gobcore/quality/issue.py:285]: Don't think this is necessary anymore

**Review by JKoetsier — APPROVED** [review]

**Pull request merged**